Stock.h

```diff
@@ -16,8 +16,7 @@ class Stock{
     Stock();
     void buyShares(int numShares);
     void sellShares(int numShares);
-    virtual void setEODStockPrice() = 0;
-    void setMaxVariance(float maxVariance);
+    void setEODStockPrice();
```

Comment:
This is fine for now but we may want to make this function virtual later on if the calculation was to change for a bond versus a basic stock.

Reply:
Yeah, @srikoushik3 and I were talking about this yesterday. Decided to keep it as is unless a change required.